### ISSUE TITLE
kwik.cx

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#2321]
### Basic Information

**Test link:** `https://kwik.cx/f/YdxUbtTlabWH`
**Category:** anti-adblock
**Reported from:** Hong Kong
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36
**Extension:** Nano Defender 15.0.0.170

### Additional Message

Above URL won't work if directly accessed. Use the redirect link instead: `http://cowner.net/-81162TQPJ/1e8j?rndad=460983879-1574247357`.

Basically anything from `https://kwik.cx/`, a video streaming service which will hide their download button if uBlock is activated. Done that for months, until I can't handle their massive popups anymore.
